### PR TITLE
Update to reflect new major version changes

### PR DESCRIPTION
**Pull request by jugarrit**

### CODE REVIEW

**Review by mspensieri — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -75,11 +75,6 @@ Smooch.setUserProperties({
 });
 ```
 
-## Track events to show Smooch Whispers
-```js
-Smooch.track("Slapped kid");
```
Comment (wmora) [inline README.md:80]: 🤦‍♂️

**Review by wmora — APPROVED** [review]
👍

**Comment by Go-Fred** [comment]
👍

File: README.md (hunk 1 above)
Comment (Go-Fred) [reply README.md:80]: 😳

**Pull request merged**